src/weapon.py

```diff
@@ -76,44 +68,29 @@ def __init__(self, input_button: VirtualButton, car: arcade.Sprite):
         self.rocket_speed = 200
         self.fire_rate = 0.5
 
-    def update(self, delta_time):
+    def update(self, delta_time, projectile_list: arcade.SpriteList, beam_list: arcade.SpriteList):
         if not self.button_held and self.input_button.value:
             self.button_held = True
             if self.time_since_shoot > 1 / self.fire_rate:
-                self.shoot()
+                self.shoot(projectile_list)
         if self.button_held and not self.input_button.value:
             self.button_held = False
         self.update_active_weapon(delta_time)
-        self.remove_bullets()
 
-    def shoot(self):
+    def shoot(self, projectile_list: arcade.SpriteList):
         rocket = arcade.SpriteSolidColor(50, 30, arcade.color.ORANGE)
         rocket.center_x = self.car.center_x
         rocket.center_y = self.car.center_y
+        rocket_angle = math.radians(self.car.angle)
```

Comment:
I recently learned we can do `self.car.radians`.  Pretty sure we already talked about this on discord.

Reply:
Implimented